.gitignore

```diff
@@ -23,5 +23,6 @@
 *.DS_Store
 *.pyc
 .vscode/
-
+.vscode
```

Comment:
Is this really necessary with `.vscode/`?

Reply:
I didn't realize it was already there.

Reply:
Solved, I already deleted one of the two files from the gitignore